StarterProject/pageobjects/common.js

```diff
@@ -0,0 +1,87 @@
+import Page from "./page.js";
+
+export default class Common extends Page {
+
+    constructor() {
+        super();    
```

Comment:
we need to be _really_ careful with inheritance in JavaScript, it doesn't always work out as well as we'd expect in other languages.  Unless of course Page is an abstract class in which case 👍

Reply:
I don't understand the concept of inheritance in JavaScript. I am using my ruby understanding for much of this right now